lib/models/room.js

```diff
@@ -600,6 +604,30 @@ Room.prototype.addTags = function(event) {
     this.emit("Room.tags", event, this);
 };
 
+/**
+ * Update the account_data events for this room, overwriting events of the same type.
+ * @param {Array<MatrixEvent>} events an array of account_data events to add
+ */
+Room.prototype.addAccountData = function(events) {
+    for (var i = 0; i < events.length; i++) {
+        var event = events[i];
+        if (event.getType() === "m.tag") {
+            this.addTags(event);
+        }
+        this.accountData[event.getType()] = event;
+        this.emit("Room.accountData", event, this);
+    }
+};
+
+/**
+ * Access account_data event of given event type for this room
+ * @param {string} type the type of account_data event to be accessed
+ * @return {MatrixEvent} the account_data event in question
```

Comment:
Nit: Technically this should be `{?MatrixEvent}` as it is possible for this function to return `undefined`.